src/preemptive/lwt_preemptive.ml

```diff
@@ -207,20 +207,20 @@ let detach f args =
   in
   Lwt.finalize
     (fun () ->
-      (* Send the id and the task to the worker: *)
+       (* Send the id and the task to the worker: *)
        CELL.set worker.task_cell (id, task);
        waiter)
     (fun () ->
-      if worker.reuse then
-        (* Put back the worker to the pool: *)
-        add_worker worker
-      else begin
-        decr threads_count;
-        (* Or wait for the thread to terminates, to free its associated
-           resources: *)
-        Thread.join worker.thread
-      end;
-      Lwt.return_unit)
+       if worker.reuse then
```

Comment:
Hmm, I guess it does this to align everything two spaces after the `fun` keyword, which is one column after the beginning of the line. I agree it looks kinda wrong. I'll see what I can do.